web/package.json

```diff
@@ -37,6 +55,29 @@
     }
   },
   "dependencies": {
-    "axios": "^0.24.0"
+    "@types/copy-webpack-plugin": "8.0.1",
+    "axios": "0.24.0"
```

Comment:
btw in a previous PR I've put axios in dependencies instead of devDependencies, but I don't think it should be different than the other runtime stuff (like react and so on)..
Do you know why we can have the runtime deps in devDependencies? is it related to the way the console plugin works (with the webpack stuff etc.) ? (cc @OlivierCazade )

Reply:
My understanding is that unless npm is told to not install devdependencies (using --production flag), it will install both.

Currently we are installing both, and we need to switch some dependencies to hace a working build without devdepndencies.

Reply:
Yes I just took the existing thinking that the plugin doesn't need all this dependencies in prod. I can move them :+1:

Reply:
ok it doesn't have to be in this PR, but we should do that +1